src/Admin.php

```diff
@@ -262,4 +262,10 @@ public static function admin_head() {
 EOD;
   }
 
+  /**
+   * Disables link notifications from other web sites.
+   */
+  public static function disableLinkNotifications() {
+    update_option('default_ping_status', 'closed');
```

Comment:
This should be implemented as a filter for the option name instead.

(Aside from that your proposed code would perform a write operation to the options table on every single request to an admin page – never do something like that.)

Reply:
1. Ok, I will try the filter approach.
2. I am calling this function from the plugin activation hook. Is this not right then?

Reply:
Sorry about 2., I wrongly assumed a different invocation.

Reply:
Accepting 2 as correct, I feel I should not set `default_ping_status` option with a filter, as it should be triggered in every frontend page rendering. Also, I find it somehow confusing having that option 'checked' in the site configuration but doing the opposite in the backstage.

Reply:
The option should be forced to be disabled / not allow pingbacks.  The override should also affect existing sites.

The stored option value will not be visible in the backend, because the value will be always overridden.  If you want to prevent confusion, you can additionally disable/grey-out the option in the backend UI – which AFAIK should be easy to do, as all options are registered as admin settings and their form elements can be filtered individually.